robottelo/cli/factory.py

```diff
@@ -1717,14 +1717,19 @@ def setup_org_for_a_custom_repo(options=None):
             )
         except CLIReturnCodeError as err:
             raise CLIFactoryError(f'Failed to associate activation-key with CV\n{err.msg}')
-    # Add subscription to activation-key
-    activationkey_add_subscription_to_repo(
-        {
-            'activationkey-id': activationkey_id,
-            'organization-id': org_id,
-            'subscription': custom_product['name'],
-        }
-    )
+
+    # Add custom_product subscription to activation-key, if SCA mode is disabled
+    if (
+        Org.info({'fields': 'Simple content access', 'id': org_id})['simple-content-access']
```

Comment:
@Gauravtalreja1  good idea to create a helper function for checking if org has sca enabled or not?

Reply:
@jameerpathan111 Thanks for the suggestion, updated